compute/compute-node.Dockerfile

```
@@ -1778,11 +1778,14 @@ COPY --from=pg_partman-src /ext-src/ /ext-src/
 #COPY --from=pg_repack-src /ext-src/ /ext-src/
 
 COPY --chmod=755 docker-compose/run-tests.sh /run-tests.sh
+RUN rm -f /ext-src/*.tar.gz
+RUN apt-get update && apt-get install -y libtap-parser-sourcehandler-pgtap-perl
```

Can we merge these two commands and clean the cache aftewards?
```suggestion
RUN apt-get update && apt-get install -y libtap-parser-sourcehandler-pgtap-perl\
   && apt clean && rm -rf /ext-src/*.tar.gz /var/lib/apt/lists/*
```

---

BTW, I wanted to comment the same because it's usually better to have less layers, but then I thought that it's not a good idea because `/ext-src/*.tar.gz` will change more often, which will prevent this common layer caching

Is it right? Or I misinterpret that? @myrrc

---

If we split back, layers won't be cached as well as newer layers depend on older ones (i.e. ext-src removal) so they would be rebuilt anyway.

---

We can move libtap-parser-sourcehandler-pgtap-perl installation to the top of the image for it to be cached

---

Anyway, we use the cache from the previous step [source](https://github.com/neondatabase/neon/blob/a4e3989c8da1bf0dc8a88b35a010055c29afd43f/.github/workflows/build_and_test.yml#L662)